app/views/widgets/ad_landing_page/_masthead.html

```diff
@@ -0,0 +1,36 @@
+<div class="ad-landing-masthead">
+  <div class="masthead-header">
+    <%= image_tag("argyros-center-stacked.svg", class: "header-mobile") %>
+    <div class="header-desktop">
+      <%= image_tag("argyros-horizontal-stacked.svg", class: "header-logo-desktop") %>
+      <p class="header-text-desktop">MAKING <strong>BOLD</strong> STEPS</p>
+    </div>
+  </div>
+
+  <div class="masthead-hero">
+    <div class="background-image" style="background-image: url('http://www.chapman.edu/dodge/_files/dmac/dmac-masthead.jpg'); ">
+      <div class="outer-background-overlay"></div>
+      <article style="background-image: url('http://www.chapman.edu/dodge/_files/dmac/dmac-masthead.jpg'); ">
+        <div class="inner-background-overlay"></div>
+        <div class="content">
+          <h1 class="title">TURN YOUR PASSION, KNOWLEDGE AND EXPERIENCE INTO A REWARDING CAREER</h1>
+          <h2 class="subtitle">Earn your Masters Degree at Dodge College of Film and Media Arts.</h2>
+        </div>
+      </article>
+    </div> 
+  </div>
+
+  <div class="masthead-content">
+    <section class="masthead-content-panel">
+      <h2>Take the first step to advance your career.</h2>
+
+      <div class="calls-to-action">
+        <div id="form_a4517720-e7d7-4e0e-bafd-2c1045918e0d">Loading...</div><script>/*<![CDATA[*/var script = document.createElement('script'); script.async = 1; script.src = 'https://go.chapman.edu/register/?id=a4517720-e7d7-4e0e-bafd-2c1045918e0d&output=embed&div=form_a4517720-e7d7-4e0e-bafd-2c1045918e0d' + ((location.search.length > 1) ? '&' + location.search.substring(1) : ''); var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(script, s);/*]]>*/</script>
```

Comment:
Is this the form code? Let's add comment explaining what it does.

Break js into multiple lines if possible.

Also, why CDATA wrapper for js?